pom.xml

```diff
@@ -129,6 +130,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-spring-boot-bom</artifactId>
```

Comment:
Can you add it in the spring boot module? It may cause issues with ceq (at least upstream)

Reply:
Just a dependencyManagement entry and not import scope - I think we're okay here.   Not adding a dependency, it's just overriding the version of org.apache.camel.springboot:camel-spring-boot-bom if it is used to ${camel-spring-boot-version}.     Shouldn't conflict with CEQ at all because they won't have that bom as a dependency anywhere.